layers/+lang/haskell/config.el

```diff
@@ -17,7 +17,7 @@
 (spacemacs|defvar-company-backends haskell-cabal-mode)
 (spacemacs|defvar-company-backends intero-repl-mode)
 
-(defvar haskell-completion-backend 'ghci
+(defvar haskell-completion-backend `ghci
```

Comment:
thanks, I think they work all the same (I didn't have any issues on this case) but, as you said, the convention seems to be the backticks(`)  are used only when it's needed specifically. (as in the `pcase` use here)